### ISSUE TITLE
Official Mastodon Ansible playbook not documented as a deployment option

### ISSUE THREAD

**Issue by thunderysteak** [mastodon/documentation#1053]
The Mastodon Ansible Playbook is a quick and easy way of deploying and patching Mastodon instances and looks like not a lot of people are aware of it.

https://github.com/mastodon/mastodon-ansible

It works on Ubuntu 20.04, Ubuntu 21.04, RHEL 8 derivatives and RHEL 9 derivatives with builds passing

I feel like this should be included in the docs

**Comment by ineffyble** [comment]
I think it's recently been uplifted and having active development again, so it's probably a good idea to add it to the docs, yeah.

**Comment by mjankowski** [comment]
In the time since this was first made, the ansible playbook appears to now be not very actively maintained anymore -- no commits in 18 months, and some unresponded to issues/PRs. Given that, not sure inclusion in docs makes sense.

**Comment by thunderysteak** [comment]
Yeah I got too busy IRL to contribute to it and nobody really picked up the work and tried to fix it for the newer mastodon release. Even asked for call to action for people to help on a Mastodon meetup in September with no results. 

Since the Ansible playbooks are unfortunately abandoned by both the community & staff I'm closing this
